### ISSUE TITLE
[Bug][Scheduler] The aria-label of the item does not contain the end date for the event

### ISSUE THREAD

**Issue by Yanko-Georgiev** [telerik/kendo-react#3464]
### Description

When an event spans across multiple days, you would expect the aria-label to contain the end date as well. For example, if an event is from January 1 2025 to December 31 2025, the aria-label should be `Wednesday, January 1, 2025, 8:00 AM-Wednesday, December 31, 2025, 8:30 PM, Event Name`. However, currently, the aria-label does not contain the end date `Wednesday, January 1, 2025, 12:00 AM–2:00 AM, Event Name`. This is only correct if the event ends on the same day.

### Plan

KendoReact

### Steps To Reproduce

Open the example and inspect the "Bowling tournament" item: https://stackblitz.com/edit/react-28qmx6kv-v3anbeny?file=app%2Fapp.tsx

### Screenshots or video

<img width="1915" height="674" alt="Image" src="https://github.com/user-attachments/assets/883bdeec-3e5a-4dc4-8668-dd99eef8841d" />

### Actual Behavior

The aria label does not specify the end date.

### Expected Behavior

If the event starts and ends on January 1 - `Wednesday, January 1, 2025, 12:00 AM–2:00 AM, Event Name`
If the event starts on January 1 but ends on another date - `Wednesday, January 1, 2025, 8:00 AM-Wednesday, December 31, 2025, 8:30 PM, Event Name`

Reported in TicketID: 1704411

**Comment by PekoPPT** [comment]
The bug is fixed. The fix will be available in the next KendoReact official release we plan to provide by the end of the week.
